mw-webapp/src/router/useValidatedParams.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {PropsWithChildren, ReactElement} from "react";
+import {useNavigate, useParams} from "react-router-dom";
+import {pages} from "src/router/pages";
+
+/**
+ * Query param types
+ */
+export type QueryParamTypes = {
+
+  /**
+   * Page's uuid
+   */
+  uuid: string;
+}
+
+/**
+ * If query params exist allow redirect to Page and if not navigate to Page404
+ */
+export const ValidatedParams = (props: PropsWithChildren) => {
+  const {uuid} = useParams<QueryParamTypes>();
```

Comment:
does every page shpould has validate params? Good prototype, but it does not look reusable

Reply:
Changed validated wrapper but also dont know how to make it reusable(((